README.md

```diff
@@ -313,8 +319,9 @@ func.call(obj, val)
 **Babylon Label**: [Spec: BigInt](https://github.com/babel/babylon/issues?utf8=%E2%9C%93&q=label%3A%22Spec%3A%20BigInt%22%20)  
 <details>
 <summary>Code Example</summary>
-```js
-1n; // integer
+
+```
```

Comment:
@Kovensky yeah prob a good idea, I initially removed it cuz I incorrectly assumed the `js` to be responsible for broken formatting, but it was the absence of newline